### PR TITLE
More css fixes and refactors

### PR DESCRIPTION
**Pull request by Trombach**
I've made some css refactors that streamline the homepage css a little bit. The homepage should now look the same as it did before the css optimisations

### CODE REVIEW

File: research-hub-web/src/app/components/home/browse/browse.component.spec.ts [hunk 1]
```diff
@@ -5,7 +5,9 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { ApolloTestingModule } from 'apollo-angular/testing';
 import { CategoryCollection, AllCategoriesGQL } from '@graphql/schema';
 import { Observable, of } from 'rxjs';
-import { MockProvider } from 'ng-mocks';
```
Comment (robmint) [inline research-hub-web/src/app/components/home/browse/browse.component.spec.ts:8]: Looks like we have a mix of CSS fixes and test fixes?

Comment (Trombach) [reply research-hub-web/src/app/components/home/browse/browse.component.spec.ts:8]: I refactored some things in a more angular-y way, and created new components. Those needed to be mocked in the old tests.

File: research-hub-web/src/app/components/home/browse/browse.component.scss [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
  * Custom Text
  */
-.text {
-  width: 100%;
-  @extend .text;
-  @include size(large) {
-    text-align: left;
-  }
-}
+// .text {
```
Comment (rosemcc) [inline research-hub-web/src/app/components/home/browse/browse.component.scss:8]: Can commented out stuff be removed now?

File: research-hub-web/src/app/components/home/home.component.scss [hunk 3]
```diff
@@ -45,37 +34,55 @@
...
-    .standard-button:hover {
-      color: $dark-grey !important;
-      border-radius: 50px !important;
-      background-color: #eeeeee !important;
-      transition: background-color 0.1s ease-in-out, color 0.1s ease-in-out;
-    }
   }
 }
+
+/**
```
Comment (rosemcc) [inline research-hub-web/src/app/components/home/home.component.scss:51]: commented out stuff - remove?

**Review by rosemcc — APPROVED** [review]
Nice Lukas, looks like those css issues have all been addressed and things look tidier too. Happy for you to merge

File: research-hub-web/src/app/components/home/home.component.scss (hunk 3 above)
Comment (Trombach) [reply research-hub-web/src/app/components/home/home.component.scss:51]: Yes, I tried to find all instances where I left comments, but it seems like I have missed a few :)

File: research-hub-web/src/app/components/home/browse/browse.component.spec.ts (hunk 1 above)
Comment (robmint) [reply research-hub-web/src/app/components/home/browse/browse.component.spec.ts:8]: All good

**Review by robmint — APPROVED** [review]

**Pull request merged**